src/Utils/Value.php

```diff
@@ -147,7 +149,7 @@ public static function coerceValue($value, InputType $type, ?VariableDefinitionN
                 ? $coercedItem
                 : self::ofValue([$coercedItem['value']]);
         }
-
+        /** @var InputObjectType $type we handled all other cases at this point */
```

Comment:
we could put `assert` here as well.

"we handled all other cases at this point" we know this know but it is error-prone for the future.